cmd/kraft/build/build.go

```diff
@@ -152,7 +152,7 @@ func (opts *Build) pull(ctx context.Context, project app.Application, workdir st
 					packmanager.WithName(opts.project.Template().Name()),
 					packmanager.WithTypes(unikraft.ComponentTypeApp),
 					packmanager.WithVersion(opts.project.Template().Version()),
-					packmanager.WithCache(!opts.NoCache),
```

Comment:
shouldn't all of these also have an alias to `update` now? And then slowly deprecate `--no-cache`

Reply:
Yes I agree. For now i didn't want to change any flags as this is more user facing. We can do this in a later step i think